cql3/Cql.g

```diff
@@ -1481,17 +1481,17 @@ usertypeLiteral returns [shared_ptr<cql3::user_types::literal> ut]
     : '{' k1=ident ':' v1=term { m.emplace(std::move(*k1), std::move(v1)); } ( ',' kn=ident ':' vn=term { m.emplace(std::move(*kn), std::move(vn)); } )* '}'
     ;
 
-tupleLiteral returns [shared_ptr<cql3::tuples::literal> tt]
-    @init{ std::vector<shared_ptr<cql3::term::raw>> l; }
-    @after{ $tt = ::make_shared<cql3::tuples::literal>(std::move(l)); }
-    : '(' t1=term { l.push_back(t1); } ( ',' tn=term { l.push_back(tn); } )* ')'
+tupleLiteral returns [cql3::expr::expression tt]
+    @init{ std::vector<cql3::expr::expression> l; }
+    @after{ $tt = cql3::expr::tuple_constructor{std::move(l)}; }
```

Comment:
Why is the name "tuple_constructor" better than just "tuple"?

Reply:
This expression constructs a tuple from several subexpressions. Many other expressions can yield a tuple, for example bind_variable or column_value. I'll add the explanation to the changelog.

Reply:
Done